app/controllers/documents_controller.rb

```diff
@@ -0,0 +1,19 @@
+class DocumentsController < ApplicationController
+  def create
+    @study = Study.find(params[:study_id])
+    @document = Document.new(document_params)
+    @document.study = @study
+    if @document.save
+      redirect_to @study, notice: "Document created successfully"
+    else
+      flash[:alert] = "Sorry, looks like we're missing something, can you double check?"
+      render 'studies/show'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.